pyproject.toml

```diff
@@ -47,7 +96,6 @@ write_to = "astropy/_version.py"
     line_length = 100
     known_third_party = ["erfa", "PyYAML", "packaging", "pytest", "scipy", "matplotlib"]
     known_first_party = ["astropy"]
-    multi_line_output = 4
```

Comment:
@nstarman I had to remove this part of the `isort` configuration in order to prevent `black` from fighting each other. Is there an alternate way to handle this?

Reply:
This https://github.com/psf/black/issues/127#issuecomment-381032478 indicates that `multi_line_output` mode `8` is the output mode (which is the mode set by the `profile = "black"`) that allows `isort` and `black` to be compatible.

Reply:
Unfortunately there is no alternative. ``isort`` supports styles that black does not. We had ``multi_line_output = 4`` because it was similar to Astropy's current standard. There is another ``multi_line_output`` mode that could work: **9** in https://pycqa.github.io/isort/docs/configuration/multi_line_output_modes.html

Reply:
Not sure it's any better...

Reply:
Just fewer lines

Reply:
Here is the difference between the two settings: https://github.com/WilliamJamieson/astropy/compare/black-config..black-config-alternate-multiline.

I personally prefer the current (mode 8) as I think all the additional boiler-plate seems excessive. Thoughts?

Reply:
Agreed, though it's one of the few things I don't enjoy about black.

Reply:
Well, black is in so this discussion is moot.